force-app/main/default/classes/AffiliationsWithPrgrmEnrolVMapr_TEST.cls

```diff
@@ -0,0 +1,217 @@
+        PicklistEntryService.instance = new STUB_PicklistEntryService();
+        AffiliationsWithProgramEnrollmentService.instance = new STUB_AffiliationsWithProgramEnrollmentService();
+
+        AffiliationsWithProgramEnrollmentVMapper affiliationsWithProgramEnrollmentVMapperInstance = AffiliationsWithProgramEnrollmentVMapper.getInstance();
+
+        Test.startTest();
+        AffiliationsWithProgramEnrollmentVModel affiliationsWithProgramEnrollmentVModel = affiliationsWithProgramEnrollmentVMapperInstance.getAffiliationsWithProgramEnrollmentVModel();
+        Test.stopTest();
+
+        System.assertEquals(
+            'Test Active Status 1',
+            affiliationsWithProgramEnrollmentVModel.role.value,
+            'Selected Affiliations With Program Enrollment role should match value in hierarchy settings.'
+        );
+
+        System.assertEquals(
+            3,
+            affiliationsWithProgramEnrollmentVModel.role.options.size(),
+            'Should only include active Statuses and None as Affiliations With Program Enrollment Role options.'
+        );
+
+        System.assertEquals(
+            'Test Active Status 2',
+            affiliationsWithProgramEnrollmentVModel.status.value,
+            'Selected Affiliations With Program Enrollment Status should match value in hierarchy settings.'
+        );
+
+        System.assertEquals(
+            3,
+            affiliationsWithProgramEnrollmentVModel.status.options.size(),
+            'Should only include active Statuses and None as Affiliations With Program Enrollment Status options.'
+        );
+
+        System.assertEquals(
+            false,
+            affiliationsWithProgramEnrollmentVModel.copyEndDate,
+            'Affiliations With Program Enrollment end date should not be enabled.'
+        );
+
+        System.assertEquals(
+            false,
+            affiliationsWithProgramEnrollmentVModel.copyStartDate,
+            'Affiliations With Program Enrollment start date should not be enabled.'
+        );
+    }
+
+    /***********************************************************************************************************************************
+     ****************************************************** STUBS ***********************************************************************
+     ***********************************************************************************************************************************/
+
+    /**************************************************************************************************************************************
+     * @description Stub class to simulate the response from PicklistEntryService to return a valid respose
+     **************************************************************************************************************************************/
+    private class STUB_PicklistEntryService extends PicklistEntryService {
```

Comment:
@MatthewBlanski I am using only 1 stub for 2 methods calls, is this ok?

Reply:
This should be fine. Stub where stubs are needed for unit testing.